.cirun.yml

```diff
@@ -0,0 +1,31 @@
+# Self-Hosted Github Action Runners on AWS via Cirun.io
+# Reference: https://docs.cirun.io/Reference/yml.html
+runners:
```

Comment:
Does this mean the CI will run on Kong's AWS ec2?

Reply:
It does. I tried with docker emulation on github action runner and it doesn't have enough cpu/ram :(